src/dataMap/metaManager/metaSchema.js

```diff
@@ -1019,6 +1019,8 @@ export default () => {
 
     /**
```

Comment:
I just realized that the page https://handsontable.com/docs/8.2.0/Options.html is wrongly formatted:

<img width="1732" alt="Screenshot 2020-12-09 at 12 55 35" src="https://user-images.githubusercontent.com/566463/101626901-e11a6c80-3a1d-11eb-97c1-4051bbef5f57.png">

Reply:
I believe this is fixed with 766c56bc47b27b4f90eaf4f26b643a4fe7bce89e